pheps/phep-9999.md

```diff
@@ -0,0 +1,162 @@
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
+| :-: | :----------: | :-----------: | :------------: | :-------------: |
+| Self-Evaluation Status | Completed | Completed | Completed | Not Done |
+| PyHC Standard Grades | Mostly green, some yellow | Several yellow, no red | A couple red | N/A |
+| pyOpenSci Review Status | Completed | In progress | Not Started | Not Started |
+| PyHC Env Installation Conflicts | No conflicts | A couple conflicts exist | Major conflicts exist | Major conflicts exist |
+| HSSI Metadata Compliant | Fully Compliant | A couple issues exist | Major issues exist | Major issues exist |
+| Software License | Fully compliant and excludes NOSA license | Fully compliant, allows NOSA license | Has non-recommended license (e.g., GPL) | Has no license | 
+| DOI | Required | Required | Required | Not Required |
+| pip installable? | Yes | Yes | Yes | No |
+| conda installable? | Yes | No | No | No |
+
+Descriptions for each heading are as follows:
+   - Self Evaluation Status: indicates whether a package has completed a self evaluation against PyHC's standards
+   - PyHC Standard Grades: indicates status of each standards grouping within a package's self evaluation
+   - pyOpenSci Review Status: indicates status of a pyOpenSci review
+   - PyHC Env Installation Conflicts: indicates state of installation conflicts within the PyHC software environment (i.e. does the Package comply with PHEP 3)
+   - HSSI Metadata Compliant: indicates state of compliance with the new Heliophysics Software Search Interface's metadata requirements
+   - Software License: indicates level of software license compliance with the [PyHC Standards Document](https://doi.org/10.5281/zenodo.2529131) 
+   - DOI: indicates whether or not a package's software repository has a DOI (e.g., from Zenodo) 
+   - pip installable?: indicates whether a package is pip installable
+   - conda installable?: indicates whether a package is conda installable
+
+The following table shows the benefits that are associated with each tier:  
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
+| :-: | :----------: | :-----------: | :------------: | :-------------: |
+| Summer School Inclusion * | Yes | Yes** | No | No |
+| PyHC-top-tier Env Inclusion * | Yes | Yes ** | No | No | 
+| PyHC-all Env Inclusion * | Yes | Yes | Yes | No |
+| PyHC-Chat Bot Inclusion * | Yes | Yes ** | No | No |
+| pyOpenSci Verified Badge | Yes | No | No | No |
+| Standards Compliance Assistance * | Yes *** | Yes | Yes | Yes |
+| Main Page Listing | Yes | Yes | Yes | No |
+| Secondary Page Listing | No | No | No | Yes |
+| HSSI Inclusion | Yes | Yes | Yes | Yes | 
+| Conference Travel Funding | Yes | Yes | Yes | No |  
+
+*  = can opt-in or opt-out of this  
```

Comment:
This asterisk gets rendered as a bullet point. I think you have to escape it with a backslash.

Reply:
womp. Thanks for the note, I'll add that in.